src/backend/joanie/payment/__init__.py

```diff
@@ -16,3 +17,22 @@ def get_payment_backend():
             "JOANIE_PAYMENT_BACKEND configuration seems not valid. "
             "Check your settings.py."
         ) from error
+
+
+def get_country_calendar():
+    """
+    Instantiate the contract's calendar through `JOANIE_CONTRACT_COUNTRY_CALENDAR` setting.
+    """
+    calendar_path = getattr(settings, "JOANIE_CONTRACT_COUNTRY_CALENDAR", None)
+    if not calendar_path:
+        raise ValueError(
+            "`JOANIE_CONTRACT_COUNTRY_CALENDAR` setting must be defined in your settings.py."
+        )
+    try:
+        return import_string(calendar_path)()
+    except (AttributeError, ImportError) as error:
+        raise ImproperlyConfigured(
+            "Cannot instantiate a calendar. "
+            "`JOANIE_CONTRACT_COUNTRY_CALENDAR` configuration seems not valid. "
+            "Check your settings.py"
+        ) from error
```

Comment:
I think you could simplify this code like this
```suggestion
    try:
        calendar_path = getattr(settings, "JOANIE_CONTRACT_COUNTRY_CALENDAR")
        return import_string(calendar_path)()
    except (AttributeError, ImportError) as error:
        raise ImproperlyConfigured(
            "Cannot instantiate a calendar. "
            f'`JOANIE_CONTRACT_COUNTRY_CALENDAR="{calendar_path}"` configuration seems not valid. '
            "Check your settings.py"
        ) from error
```
`getattr` default value is already None.
If `calendar_path` is empty,  calling `import_string` with it  will raise an ImportError.
WDYT ?